src/library/trackmodel.h

```diff
@@ -29,26 +29,27 @@ class TrackModel {
     // These enums are the bits in a bitvector. Any individual column cannot
     // have a value other than 0, 1, 2, 4, or 8!
     enum Capabilities {
-        TRACKMODELCAPS_NONE              = 0x00000,
-        TRACKMODELCAPS_REORDER           = 0x00001,
-        TRACKMODELCAPS_RECEIVEDROPS      = 0x00002,
-        TRACKMODELCAPS_ADDTOPLAYLIST     = 0x00004,
-        TRACKMODELCAPS_ADDTOCRATE        = 0x00008,
-        TRACKMODELCAPS_ADDTOAUTODJ       = 0x00010,
-        TRACKMODELCAPS_LOCKED            = 0x00020,
-        TRACKMODELCAPS_EDITMETADATA      = 0x00040,
-        TRACKMODELCAPS_LOADTODECK        = 0x00080,
-        TRACKMODELCAPS_LOADTOSAMPLER     = 0x00100,
-        TRACKMODELCAPS_LOADTOPREVIEWDECK = 0x00200,
-        TRACKMODELCAPS_REMOVE            = 0x00400,
-        TRACKMODELCAPS_RESETPLAYED       = 0x02000,
-        TRACKMODELCAPS_HIDE              = 0x04000,
-        TRACKMODELCAPS_UNHIDE            = 0x08000,
-        TRACKMODELCAPS_PURGE             = 0x10000,
-        TRACKMODELCAPS_REMOVE_PLAYLIST   = 0x20000,
-        TRACKMODELCAPS_REMOVE_CRATE      = 0x40000,
+        TRACKMODELCAPS_NONE              = 0x00000u,
+        TRACKMODELCAPS_REORDER           = 0x00001u,
+        TRACKMODELCAPS_RECEIVEDROPS      = 0x00002u,
+        TRACKMODELCAPS_ADDTOPLAYLIST     = 0x00004u,
+        TRACKMODELCAPS_ADDTOCRATE        = 0x00008u,
+        TRACKMODELCAPS_ADDTOAUTODJ       = 0x00010u,
+        TRACKMODELCAPS_LOCKED            = 0x00020u,
+        TRACKMODELCAPS_EDITMETADATA      = 0x00040u,
+        TRACKMODELCAPS_LOADTODECK        = 0x00080u,
+        TRACKMODELCAPS_LOADTOSAMPLER     = 0x00100u,
+        TRACKMODELCAPS_LOADTOPREVIEWDECK = 0x00200u,
+        TRACKMODELCAPS_REMOVE            = 0x00400u,
+        TRACKMODELCAPS_RESETPLAYED       = 0x02000u,
+        TRACKMODELCAPS_HIDE              = 0x04000u,
+        TRACKMODELCAPS_UNHIDE            = 0x08000u,
+        TRACKMODELCAPS_PURGE             = 0x10000u,
+        TRACKMODELCAPS_REMOVE_PLAYLIST   = 0x20000u,
+        TRACKMODELCAPS_REMOVE_CRATE      = 0x40000u,
```

Comment:
Can we make this an enum class and use QFlags for this? (We can also do this in another PR)

Also, for bitfields I usually prefer this notation over hex, because it becomes much easier to spot mistakes:

    0
    1
    1 << 1
    1 << 2
    1 << 3

Reply:
Yes we can do that, but please hold on with such functional changes - this is supposed to be a pure refactoring PR. The more we change, the larger the chance for conflicts with #2717, and I don't fancy that.

Reply:
okay.

Reply:
Fixed in #2810 :)